package.json

```diff
@@ -15,13 +15,14 @@
   "main": "platform/node/index.js",
   "repository": {
     "type": "git",
-    "url": "git://github.com/maplibre/maplibre-gl-native.git"
+    "url": "git+https://github.com/maplibre/maplibre-gl-native.git"
   },
   "license": "BSD-2-Clause",
   "dependencies": {
     "@mapbox/cmake-node-module": "^1.2.0",
+    "@mapbox/node-pre-gyp": "^1.0.9",
+    "@acalcutt/node-pre-gyp-github": "1.4.8",
```

Comment:
Did you attemt to upstream these changes, or would they not make sense for anyone but us?

Reply:
Two changes were specific to us, and may not be useful to others upstream

1.) The original package targets the 'master' branch and has no real way to change or specify a different branch
https://github.com/acalcutt/node-pre-gyp-github/blob/master/index.js#L71

2.) I changed the default name to include 'node-v' instead of just 'v' so it fit in with the rest of our releases
https://github.com/acalcutt/node-pre-gyp-github/blob/master/index.js#L72

The 3rd change could be useful to others, but it specifically tagets the node version preid I set in the workflow. other users of this might want a different preid which would make this not as useful to them.

3.) Make a pre-release on versions that include "pre"
https://github.com/acalcutt/node-pre-gyp-github/blob/master/index.js#L64
https://github.com/acalcutt/node-pre-gyp-github/blob/master/index.js#L75

So ya, I don't know if these are useful upstream without some config options to make it more useful to others